draft-ietf-quic-tls.md

```diff
@@ -1116,9 +1125,22 @@ TLS KeyUpdate message.  Endpoints MUST treat the receipt of a TLS KeyUpdate
 message as a connection error of type 0x10a, equivalent to a fatal TLS alert of
 unexpected_message (see {{tls-errors}}).
 
-An endpoint MUST NOT initiate more than one key update at a time.  A new key
-cannot be used until the endpoint has received and successfully decrypted a
-packet with a matching KEY_PHASE.
+An endpoint MUST NOT initiate the first key update until the handshake is
+confirmed from its perspective. An endpoint MUST NOT initiate a subsequent
```

Comment:
```suggestion
confirmed complete. An endpoint MUST NOT initiate a subsequent
```

Reply:
The idea here is that we want the handshake to be confirmed, not just complete

Reply:
This leans less on the definition of a new term and more on the text I suggested before.  This is the one place you rely on that term, so I would say that a strictly defined term isn't necessary, particularly one as long as "handshake is confirmed from its perspective".

Reply:
I think the shorter term with a reference reads well now